### DISCUSSION TITLE
NUCLEO-H7S3L8 board bring up

### DISCUSSION THREAD

**Discussion by rickrogers-ionq** [zephyrproject-rtos/zephyr#81231] [Q&A]
Hello,  I am trying to build the hello world sample application for NUCLEO-H7S3L8 board but not having any luck running the application on the board.  I copied and modified the stm32h7s78_dk board directory (see attached).  Does anyone have a working configuration for this board?

[nucleo_h7s3l8.TGZ](https://github.com/user-attachments/files/17704216/nucleo_h7s3l8.TGZ)

**Comment by FRASTM** [comment]
Hi @rickrogers-ionq 
I have one branch for that   https://github.com/FRASTM/zephyr/tree/nucleo_h7s3l8

**Comment by rickrogers-ionq** [comment]
thanks @FRASTM !  hopefully I'll have some time to try it out soon.

**Comment by FRASTM** [comment]
please see the new https://github.com/zephyrproject-rtos/zephyr/pull/81651
